src/page.ts

```diff
@@ -85,6 +94,7 @@ export class Page extends EventTarget {
     url: string | undefined,
     ws: WebSocket,
     browser: Browser,
+    options?: SandboxOptions,
```

Comment:
Couldn't this just be
```suggestion
    options: SandboxOptions,
```